cfme/tests/openstack/test_host_power_control.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+from utils import testgen
+from cfme.web_ui import Quadicon
+from cfme.infrastructure.host import Host
+
+pytest_generate_tests = testgen.generate(testgen.provider_by_type,
+                                         ['openstack-infra'], scope='module')
+
+
+@pytest.mark.usefixtures("setup_provider_modscope")
+def test_host_power_off(provider):
```

Comment:
@psav in case I am implementing prerequisite would it return to the test it skipped if it failed on prerequisite criteria? for example host is off, skipped to test host on, test on passed, could it return test host off then?